### PR TITLE
Send A DM for blocklist violation

### LINKED ISSUE #97: Send autodeleted messages in DMs to the user
Some people have been typing out long, long paragraphs about some things that have contained some words some people don't like, and they have been autodeleted by trup. However, because of the way it works now, you cannot edit out the words and resend it. For example, sending the message "foobar" gets deleted by trup, and then trup sends a message like "Please do not say 'foo' in r/unixporn. Your message: foobar" to the user who originally posted the message in DMs.

### PR DESCRIPTION
**Pull request by Tanish2002**
Please check the phrasing and suggest changes
Preview: [here](https://i.fiery.me/9v00O.png)
Fixes: #97

### CODE REVIEW

**Comment by eepykate** [comment]
How about

```
Your message has been deleted for containing a blocked word (<word>)

> message
```

**Comment by Tanish2002** [comment]
> How about
> 
> ```
> Your message has been deleted for containing a blocked word (<word>)
> 
> > message
> ```

Sure

**Comment by mlvzk** [comment]
The message could have multiple lines, I think it's better to put it in an embed

**Comment by eepykate** [comment]
@mlvzk FWIW you can do multi-line quotes, just prepend a `> ` to each line

**Comment by mlvzk** [comment]
I guess, either format is fine

File: main.go [hunk 1]
```diff
@@ -353,7 +353,17 @@ func runMessageFilter(s *discordgo.Session, m *discordgo.MessageCreate) (deleted
 	}
 
 	if matchedString != "" {
-		err := s.ChannelMessageDelete(m.ChannelID, m.ID)
+		userChannel, err := s.UserChannelCreate(m.Author.ID)
+
+		if err != nil {
+			log.Printf("Error Creating a User Channel Error: %s\n", err)
+		} else {
+			_, err := s.ChannelMessageSend(userChannel.ID, fmt.Sprintf("Your Message was deleted Because of a Blocklist Violation `%s`\n Please Edit it and resend it\n ```\n%s\n```", matchedString, m.Content))
```
Comment (buffet) [inline main.go]: Lowercase Because and Edit and probably also blocklist violation.

File: main.go [hunk 2]
```diff
@@ -353,7 +353,17 @@ func runMessageFilter(s *discordgo.Session, m *discordgo.MessageCreate) (deleted
...
+			if err != nil {
+				log.Printf("Error Sending a DM\n")
```
Comment (buffet) [inline main.go]: Lowercase Sending

**Review by buffet — CHANGES_REQUESTED** [review]

File: main.go (hunk 2 above)
Comment (Tanish2002) [reply main.go]: done

**Review by mlvzk — APPROVED** [review]

**Comment by mlvzk** [comment]
Thanks

**Pull request merged**